src/v/raft/consensus.cc

```diff
@@ -136,21 +137,25 @@ consensus::consensus(
 }
 
 void consensus::setup_metrics() {
+    namespace sm = ss::metrics;
+
     if (config::shard_local_cfg().disable_metrics()) {
         return;
     }
 
     _probe.setup_metrics(_log.config().ntp());
     auto labels = probe::create_metric_labels(_log.config().ntp());
-    namespace sm = ss::metrics;
+    auto aggregate_labels = ssx::aggregation_labels(
+      ssx::aggregate_partitions::no);
```

Comment:
I don't prefer this over the previous explicit method, now I can't see what _is_ being aggregated.

Reply:
I don't feel to strongly. I prefer the older version slightly, but I don't mind.